src/main/java/core/basesyntax/writer/FileWriterImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.writer;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String resultingReport, String filePath) {
+        try (BufferedWriter bufferedWriter =
+                    new BufferedWriter(
+                            new java.io.FileWriter(
+                                    new File(filePath)
+                            )
+                    )
+        ) {
+            bufferedWriter.write(resultingReport);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
The exception message should be more informative. Include details such as the file path and the operation that failed.